src/senaite/core/content/senaitesetup.py

```diff
@@ -78,9 +78,25 @@ class ISetupSchema(model.Schema):
         required=False,
     )
 
+    immediate_results_entry = schema.Bool(
+        title=_(u"Immediate results entry"),
+        description=_(
+            "Allow the user to directly enter results after sample "
```

Comment:
Same comment about translation approach suggested above, but for `senaitesetup` type instead:

```
label=_("label_senaitesetup_immediate_results_entry", 
        default=u"Immediate results entry"),
description=_(
    "description_senaitesetup_immediate_results_entry",
    default=u"Allow the user to directly enter results after sample "
```

Reply:
:+1: